proxy/enfuse/testserver/hello.go

```diff
@@ -0,0 +1,55 @@
+//go:build !release
+// +build !release
```

Comment:
Why the build tags? Here and in other files

Reply:
They get automatically populated by gazelle. It makes it so that the go tool wont build the package for release